src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -1,17 +1,15 @@
-package com.typesafe.sbt.packager.docker
```

Comment:
This files contains formatting changes, source changes and out-dated code. Please update and move formatting into a separate pull request.